build_projects/dotnet-host-build/build.sh

```diff
@@ -40,6 +42,9 @@ while [[ $# > 0 ]]; do
             # Allow CI to disable prereqs check since the CI has the pre-reqs but not ldconfig it seems
             export DOTNET_INSTALL_SKIP_PREREQS=1
             ;;
+        --hostbuildonly)
+            __HostBuildOnly=1
```

Comment:
Thank you for your suggestion. I am going to change it according to your opinion. But I don't know if this name is okay. How about '--build-driver-only' ? Do you have a better idea?